### ISSUE TITLE
feature request: option to save as .txt file by default

### ISSUE THREAD

**Issue by Lumiras** [uranusjr/macdown#446]
Markdown is an amazing thing as a syntax, but the .md format isn't nearly as universally supported as .txt files. You can save files from Macdown as .txt files and they behave just fine, but it would be nice to be able to set whether or not the program saves files as .md or .txt files by default.

**Comment by Jmuccigr** [comment]
Do you mean just changing the default extension?

**Comment by Lumiras** [comment]
Yes, I think it would be great just to have an option in the preferences area that lets you set the default file extension as .md or .txt. I'm aware some extra features may be unavailable when saving as .txt, but it would be nice to have the option. 

If this is already a possibility in the program, I apologize, but I don't see it.

**Comment by uranusjr** [comment]
I think we can add a dropdown and preference to let users decide what default extension to use when saving. like what Mou does.

**Comment by uranusjr** [comment]
What common extensions are there? I know

* `.md`
* `.markdown`
* `.txt`

Anything else?

**Comment by Lumiras** [comment]
Those extensions are all I can think of off the top of my head as well, and are really the ones that maintain the file as plain text. You get into other formats like .rtf and there are formatting considerations that aren't necessary for writing Markdown.

**Comment by Jmuccigr** [comment]
Agree. See also http://talk.commonmark.org/t/file-extension-for-commonmark/440

**Comment by jkotsbak** [comment]
Being able to choose .txt as standard would be very useful. The "find text in file" function on my Mac does not search any of the .md files. I can change extension on the .md files, but each time I update them with MacDown they change back. A work-around could be to change the find text function. Maybe there are someone knowing how to do that?

**Comment by yaomtc** [comment]
This is the only feature of MacDown that, if I knew how to add the option and submit a patch, I would take the time to do it. Can we get a bounty on this? I'd chip in to it, if so.

**Comment by JanX2** [comment]
@jkotsbak 
You can [add Markdown files to your Spotlight search results](https://stackoverflow.com/questions/365669/how-can-i-make-spotlight-index-markdown-files#365675).

**Comment by jkotsbak** [comment]
Thank you, but i have found a workaround. I just copy an already 
existing .txt file and renames it in Macdown.

Den 13/09/2017 04:06, skrev yaomtc:

**Comment by iptaylortechnical** [comment]
Is help still wanted on this?

**Comment by franklinyu** [comment]
@iptaylortechnical PR is always welcome.

**Comment by Lumiras** [comment]
Yes, please :D
